### ISSUE TITLE
Need to set small learning rate for DCN_offset_mask?

### ISSUE THREAD

**Issue by splinter21** [xinntao/EDVR#138]
https://github.com/chengdazhi/Deformable-Convolution-V2-PyTorch/issues/9#issuecomment-457592960

According to this,
1/10 lr->stabler for training in ImageNet VID, but no need for training in coco

Is it needed for VSR?

**Comment by tongjuntx** [comment]
i think so,dcn is not stable
